djangocms_admin_style/sass/components/_base.scss

```diff
@@ -79,13 +79,15 @@ a.btn {
     @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
     display: inline-block;
     padding: 10px 20px !important;
+    &:link:hover {
```

Comment:
@vxsx Because I have to overwrite. If I write it without link it will not take the correct hover color